recipes/slow-buffer-to-buffer-alloc-unsafe-slow/src/workflow.ts

```diff
@@ -0,0 +1,291 @@
+
+	// Process comments mentioning SlowBuffer
+	hasChanges = processSlowBufferComments(rootNode, edits) || hasChanges;
+
+	return hasChanges ? rootNode.commitEdits(edits) : null;
+}
+
+/**
+ * Process require statements for the 'buffer' module and transform SlowBuffer references
+ */
+function processRequireStatements(root: SgRoot, edits: Edit[]): boolean {
+	let hasChanges = false;
+	const requireStatements = getNodeRequireCalls(root, "buffer");
+
+	for (const statement of requireStatements) {
+		// Get the name field to check what type of pattern we're dealing with
+		const nameField = statement.field("name");
+		if (!nameField) continue;
+
+		// Handle direct variable assignment: const SlowBuffer = require('buffer').SlowBuffer
+		if (nameField.kind() === "identifier" && nameField.text() === "SlowBuffer") {
+			const valueField = statement.field("value");
+			if (valueField && valueField.kind() === "member_expression") {
+				const property = valueField.field("property");
+				if (property && property.text() === "SlowBuffer") {
+					// Replace the identifier name
+					edits.push(nameField.replace("Buffer"));
+					// Replace the property in the member expression
+					edits.push(property.replace("Buffer"));
+					hasChanges = true;
+				}
+			}
+		}
+
+		// Handle destructuring patterns: const { SlowBuffer, ... } = require('buffer')
+		if (nameField.kind() === "object_pattern") {
+			const originalText = nameField.text();
+			if (originalText.includes("SlowBuffer")) {
+				let newText = transformDestructuringPattern(originalText);
+				if (newText !== originalText) {
+					edits.push(nameField.replace(newText));
+					hasChanges = true;
+				}
+			}
+		}
+	}
+
+	return hasChanges;
+}
+
+/**
+ * Process ESM import statements for the 'buffer' module and transform SlowBuffer references
+ */
+function processImportStatements(root: SgRoot, edits: Edit[]): boolean {
+	let hasChanges = false;
+	const importStatements = getNodeImportStatements(root, "buffer");
+
+	for (const statement of importStatements) {
+		// Try to find named_imports directly in the statement
+		const namedImports = statement.find({ rule: { kind: "named_imports" } });
+		if (namedImports) {
+			const originalText = namedImports.text();
+			if (originalText.includes("SlowBuffer")) {
+				let newText = transformDestructuringPattern(originalText);
+				if (newText !== originalText) {
+					edits.push(namedImports.replace(newText));
+					hasChanges = true;
+				}
+			}
+		}
+	}
+
+	return hasChanges;
+}
+
+/**
+ * Transform destructuring patterns by replacing SlowBuffer with Buffer
+ * Handles cases like { SlowBuffer }, { Buffer, SlowBuffer }, { SlowBuffer, Buffer }, etc.
+ * Also handles aliased patterns like { SlowBuffer: SB }, { SlowBuffer: SB, Buffer }
+ * For ESM imports: { SlowBuffer as SB } -> { Buffer as SB }
+ * Preserves original formatting as much as possible
+ */
+function transformDestructuringPattern(originalText: string): string {
```

Comment:
Here it's use too much regex, we need to relie on AST for reliability. If you have difficult to visualize the ast you can use [Codemod Studio](http://app.codemod.com/studio), [ast-grep playground](https://ast-grep.github.io/playground.html) or any [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) sitter visualization tools.